### DISCUSSION TITLE
龙族

### DISCUSSION THREAD

**Discussion by Signedinn** [share121/inter-knot#4921] [灌水]
龙族手游开服了 有没有了解的讲讲这个游戏怎么样
![IMG_4640](https://github.com/user-attachments/assets/63cdca9a-748c-433d-b9be-29593443234d)

**Comment by March-777th** [comment]
鉴定为ip粉的钱真好赚

**Comment by Milekulu** [comment]
鉴定为滚服二次元皮战力游戏（）
